src/Compilers/CSharp/Test/Emit/CodeGen/UnsafeTests.cs

```diff
@@ -8217,7 +8261,7 @@ public class C
     private static unsafe int Main()
     {
         Int64* intArray = stackalloc Int64[0x7fffffff];
-        return 0;
+        return (int)intArray[0];
```

Comment:
I had to "use" the result or optimizer would take the whole thing out. That would be correct, but would defeat the purpose of the test.